datafusion/core/src/dataframe/mod.rs

```diff
@@ -2172,31 +2171,6 @@ mod tests {
         Ok(())
     }
 
-    #[tokio::test]
-    async fn select_with_window_exprs() -> Result<()> {
-        // build plan using Table API
-        let t = test_table().await?;
-        let first_row = Expr::WindowFunction(WindowFunction::new(
-            WindowFunctionDefinition::BuiltInWindowFunction(
-                BuiltInWindowFunction::FirstValue,
-            ),
```

Comment:
This test should be fixed to use `first_value_udwf()` and not deleted.

Reply:
done thanks